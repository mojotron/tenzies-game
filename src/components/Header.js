import "../styles/Header.css";

const Header = () => {
  return (
    <header className="Header">
      <h1 className="Header__heading">Tenzies</h1>
      <p className="Header__description">
        Roll until all dice are the same. Click each die to freeze it at its
        current value between rolls.
      </p>
    </header>
  );
};

export default Header;
