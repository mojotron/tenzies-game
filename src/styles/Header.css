.Header {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}
.Header__heading {
  font-family: "Bungee", "Poppins", sans-serif;
  font-size: 3.5rem;
  color: var(--yellow);
  margin-bottom: 3rem;
}
.Header__description {
  text-align: center;
  border-radius: 0.5rem;
  color: var(--black);
  padding: 2rem 3rem;
  line-height: 2.5rem;
  font-size: 1.4rem;
  background-color: var(--gray);
}
