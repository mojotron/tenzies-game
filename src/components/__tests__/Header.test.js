import { render } from "@testing-library/react";
import Header from "../Header";

describe("Header component", () => {
  test("header snapshot", () => {
    const { component } = render(<Header />);
    expect(component).toMatchSnapshot();
  });
});
