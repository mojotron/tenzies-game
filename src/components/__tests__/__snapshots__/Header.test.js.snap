// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component header snapshot 1`] = `undefined`;
